cmd/syncthing/main.go

```diff
@@ -701,7 +701,9 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 
 	dbFile := locations[locDatabase]
 	ldb, err := db.Open(dbFile)
-	if err != nil {
+	if err == db.ErrDatabaseDowngrade {
+		l.Fatalln("Cannot open database:", err, "- Did you downgrade Syncthing? If you can't upgrade to the most recent version, you need to reset the database (-reset-db) which will cause a full rehash of all your files")
```

Comment:
Wat? We can open the database... We should also mention that any out of sync files will end up being conflicts.

Reply:
Good point. Question: Will all of them become conflicts or just those, that were never updated locally? E.g. if a file before was {ourID: 10, otherID: 10}, after reset it will be {ourID: 1}, this wouldn't be a conflict. If I am not mixing this up, a less dramatic formulation like "out of sync files may end up being conflicts" is appropriate?